### PR TITLE
rebalance: handle events while rebalancing

### PR DESCRIPTION
**Pull request by gallizoltan**

### CODE REVIEW

**Comment by gallizoltan** [comment]
@m-schmoock check this out! This is what I do if an event occurs while rebalancing! 😎

**Review by m-schmoock — DISMISSED** [review]
ACK b922b05

A PR description would be nice (next time) and we usually don't like merge commits.
Anyway no prob, I also misuse Github GUI once in a while to create unwanted merges.

I was able to get rid of the merge commit :) but now CI wants to do another round :D

**Review by gallizoltan — COMMENTED** [review]
Sorry for the merge commit, I didn't know what that button does. 😇

File: rebalance/rebalance.py [hunk 1]
```diff
@@ -706,28 +709,37 @@ def feeadjuster_toggle(new_value: bool):
...
-        plugin.log(f"Automatic rebalance is running with enough liquidity threshold: {plugin.enough_liquidity.to_satoshi_str()}, "
-                   f"ideal liquidity ratio: {plugin.ideal_ratio * 100:.2f}%, "
-                   f"min rebalancable amount: {plugin.min_amount.to_satoshi_str()}, "
-                   f"feeratio: {plugin.feeratio}")
+        plugin.log(f"Automatic rebalance started")
         failed_channels = []
         success = 0
         fee_spent = Millisatoshi(0)
-        while not rebalance_stopping():
+        while True:
```
[the remark this answers is not in the document]
Comment (m-schmoock) [reply rebalance/rebalance.py:732]: alrighty then

**Pull request merged**